lib/site.taxonomy.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Module dependencies.
+ */
+import Term from './site.taxonomy.term';
+
+/**
+ * Taxonomy class
+ */
+class Taxonomy {
```

Comment:
About name convention ( do we have? Not, we don't. ) we should try use the whole path in the class name. What I mean for instance if the method will hit to `/site/$site/users` then:

  * the `js` filename should be `site.users.js`
  * the class name to uses site users endpoint should be `SiteUsers`
  * the entry method from `Site` class to create a `SiteUsers` class should be `Site#users()`

In this case:

- [x] js file: site.taxonomy.js
- [x] Site method: Site#taxonomy()
- [ ] taxonomy class name: Taxonomy -> SiteTaxonomy